content.js

```diff
@@ -2,8 +2,13 @@ if(id = getYouTubeVideoID(document.URL)){ // Direct Links
   videoIDChange(id);
 }
 
-var hotkeys = {};
-hotkeys.submit = "Semicolon";
+var Settings = {} // Needs to be Synced with global settings
+
+Settings.hotkeys = {}; // TEMP
+Settings.onekey = false; // TEMP
+Settings.hotkeys.submit = "Quote"; // TEMP
+Settings.hotkeys.start = "Semicolon"; // TEMP
```

Comment:
Can you set this up so the settings save? Right now it does nothing as it is by default just the same

Reply:
It would be there no place to change them or to store the default values

Reply:
Yea, for now you can put it in the options section of the popup

Reply:
There is in the option page currently???

Reply:
Yes there is, click the options button  the popup

Reply:
Waiting for the UI changes